pkgs/development/typst-packages/cetz.nix

```diff
@@ -0,0 +1,35 @@
+{
+  buildTypstPackage,
+  fetchFromGitHub,
+  just,
+  file,
+  perl,
+}:
+
+buildTypstPackage rec {
+  pname = "cetz";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cetz-package";
+    repo = "cetz";
+    rev = "v${version}";
+    hash = "sha256-5+Np64+0Ca8yL7D1CCUewF03Wh6BM2bV1BQNUA+bW74=";
+    deepClone = true;
+  };
+
+  nativeBuildInputs = [
+    just
+    file
+    perl
+  ];
+
+  buildPhase = ''
+    patchShebangs scripts/package
+    HOME=$(pwd) just install
+  '';
```

Comment:
Overriding entire phases seem really weird to me, especially since there are hooks for `just` that run such commands.

```suggestion
  prePatch = ''
    patchShebangs scripts/package
  '';
  
  postPatch = ''
    export HOME=$(mktemp -d)
  '';
```